.github/workflows/ttnn-post-commit.yaml

```diff
@@ -60,6 +60,8 @@ jobs:
             fast_runtime_mode_off: true
           - name: ttnn examples and cpp tests
             cmd: ./build/test/ttnn/unit_tests_ttnn && ./tests/scripts/run_ttnn_examples.sh
+          - name: ttnn ccl cpp unit tests
```

Comment:
This workflow does not run T3000 unit tests. Could you clarify your intention?

Reply:
That's fine. These tests are intended to be able to run on n300.

Reply:
Sounds good. Did you verify that it actually runs on post commit CI?

Reply:
Yessir:
<img width="248" alt="image" src="https://github.com/user-attachments/assets/61426b0d-f704-41ee-aab5-8c4aee85f63c">

Reply:
<img width="621" alt="Screenshot 2024-08-16 at 10 45 04 AM" src="https://github.com/user-attachments/assets/fecaa636-a5b6-4383-b372-f5bda156b93c">

is this what we want? This is `ttnn-unit-tests (wormhole_b0, N300) / ttnn ccl cpp unit tests wormhole_b0 N300`

Reply:
Unfortunately yes. 

I wanted to enable this as a slot to add further unit tests and a placeholder until that issue seen on CI runs is resolved and to keep the build working... though now I think of it, I guess the test should still be building regardless of these new test bucket's presence. 

Some further context for my reasoning:
This test - the same test and code - with the 15 different cases, has some reliability (hang) issues on CI for an as yet undetermined reason. However, the test test is, so far, stable across every board and machine I've tried (in total equaling billions of messages and thousands of test invocations across 5 boards). Additionally, the main component tested by these tests (EDM), was tested stable against millions of all-gather op invocations.